seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/TaskExecutionService.java

```diff
@@ -142,7 +143,7 @@ public TaskExecutionService(NodeEngineImpl nodeEngine, HazelcastProperties prope
         scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
         scheduledExecutorService.scheduleAtFixedRate(
                 this::updateMetricsContextInImap,
-                0,
+                30 * 1000, // Wait for MapStore loading to complete
```

Comment:
> Is the number too big?
> 
> @ic4y

It should be 30 seconds

Reply:
What kind of error would occur without the delay? See if the issue can be resolved by checking the status instead of using a delay.

Reply:
> What kind of error would occur without the delay? See if the issue can be resolved by checking the status instead of using a delay.

Okay, I should have added some processing, I just don't think it's necessary to schedule it immediately

Reply:
> What kind of error would occur without the delay? See if the issue can be resolved by checking the status instead of using a delay.

![image](https://user-images.githubusercontent.com/6446530/236617484-5c642c01-3873-43f0-a47c-0408ec12e0c8.png)

When MapStore is configured as initial-mode=EAGER, an exception will be thrown, and LAZY is normal, but it does not affect service operation

Reply:
You can check whether the MapStore has finished loading within the getJobMetricsBackupInterval() method. If it hasn't finished loading, just print a log and do not perform any other actions. This way, there is no need to add a delay at this point.

Reply:
> You can check whether the MapStore has finished loading within the getJobMetricsBackupInterval() method. If it hasn't finished loading, just print a log and do not perform any other actions. This way, there is no need to add a delay at this point.

Is there a specific method to check the loading status of MapStore, or is it determined by catching exceptions?